packages/core-components/src/components/DependencyGraph/DependencyGraph.tsx

```diff
@@ -394,7 +395,7 @@ export function DependencyGraph<NodeData, EdgeData>(
       ref={containerRef}
       {...svgProps}
       width="100%"
-      height={maxHeight}
+      height={minHeight}
```

Comment:
How does this affect all other usages of this component? We may need to surface this through a prop if needed, for example something like `fit?: 'grow' | 'contain'`.